release/release_test.go

```diff
@@ -177,41 +170,56 @@ func (s *ReleaseTestSuite) TestReleaseInfo(c *C) {
 }
 
 func (s *ReleaseTestSuite) TestFilesystemRootType(c *C) {
-	reported_type, err := release.FilesystemRootType()
+	output, err := exec.Command("df", "--output=fstype", "/").CombinedOutput()
```

Comment:
Am I reading this right that this calls the real `df` from the host to get the fstype of the hosts `/` ? I would prefer if we don't do this, tests may run in all sorts of containers that may prevent doing this or where there is not even a df available. I would suggest to just use a string here (again, seems like the pre-existing test was doing similar things so no blame to you for following this pattern :)

Reply:
This test in general has become obsolete. I removed it.
I created it myself in the past to ensure that the return value of the `statfs` syscall (an integer) was properly mapped to its corresponding filesystem type.

Now we no longer rely on this syscall and we get the name of the filesystem type directly, so there is no longer a need for this test to exist.